app/component/FavouriteVehicleRentalStationContainer.js

```diff
@@ -4,53 +4,53 @@ import Favourite from './Favourite';
 import { saveFavourite, deleteFavourite } from '../action/FavouriteActions';
 import { addAnalyticsEvent } from '../util/analyticsUtils';
 
-const FavouriteBikeRentalStationContainer = connectToStores(
+const FavouriteVehicleRentalStationContainer = connectToStores(
   Favourite,
   ['FavouriteStore', 'UserStore', 'PreferencesStore'],
-  (context, { bikeRentalStation }) => ({
+  (context, { vehicleRentalStation }) => ({
     favourite: context
       .getStore('FavouriteStore')
-      .isFavouriteBikeRentalStation(
-        bikeRentalStation.stationId,
-        bikeRentalStation.networks,
+      .isFavouriteVehicleRentalStation(
+        vehicleRentalStation.stationId,
+        vehicleRentalStation.network,
       ),
     isFetching: context.getStore('FavouriteStore').getStatus() === 'fetching',
     addFavourite: () => {
       context.executeAction(saveFavourite, {
-        lat: bikeRentalStation.lat,
-        lon: bikeRentalStation.lon,
-        networks: bikeRentalStation.networks,
-        name: bikeRentalStation.name,
-        stationId: bikeRentalStation.stationId,
+        lat: vehicleRentalStation.lat,
+        lon: vehicleRentalStation.lon,
+        networks: vehicleRentalStation.network,
+        name: vehicleRentalStation.name,
+        stationId: vehicleRentalStation.stationId,
         type: 'bikeStation',
       });
       addAnalyticsEvent({
-        category: 'BikeRentalStation',
-        action: 'MarkBikeRentalStationAsFavourite',
+        category: 'VehicleRentalStation',
+        action: 'MarkVehicleRentalStationAsFavourite',
```

Comment:
I think we should keep these as they were before so new analytics events have the same category as old ones.